brother_printer_fwupd/utils.py

```diff
@@ -42,7 +51,7 @@ def __init__(
         self.handler_cb = handler_cb
         self._handler = logging.StreamHandler(stream=io.StringIO())
         self._handler.setLevel(logging.DEBUG)
-        self._context = dict[str, str | bool | list[str]]()
+        self._context = dict()  # In Python 3.10+, dict[str, str | bool | list[str]]()
```

Comment:
Please consider using this syntax for dicts:
```suggestion
        self._context: typing.Dict[str, Union[str, bool, typing.List[str]] = {}
```